### PR TITLE
feat: Accept optional phrase as a password field rather than a parameter

### LINKED ISSUE #35: Accept optional phrase as a password field rather than a parameter
Currently, we are accepting the mnemonic phrase as a parameter (to init command) and this would mean, that the phrase will be visible in the user's shell/bash history. We should accept it as a password field so that it is not visible in history.

### PR DESCRIPTION
**Pull request by kayagokalp**
closes #35.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ enum Command {
     /// Initialize the HD wallet. If it is already initialized this will remove the old one.
     Init {
         #[clap(long)]
-        phrase: Option<String>,
+        import: bool,
```
Comment (mitchmindtree) [inline src/main.rs:38]: I wonder if we should instead create a dedicated `forc-wallet import` command for this?

**Review by mitchmindtree — APPROVED** [review]
Nice! One more suggestion, but doesn't need to land in this PR. Also, just a reminder that in the future we should try our best to keep `cargo update`s to dedicated PRs.

File: src/main.rs (hunk 1 above)
Comment (adlerjohn) [reply src/main.rs:38]: +1 I think it should be a different command.

Comment (kayagokalp) [reply src/main.rs:38]: Opened #37 to track that

**Pull request merged**